README.md

```diff
@@ -11,6 +11,24 @@ As part of this repo, we provide the following examples:
 ## Installation
 
 Each tool require a different installation process. TODO: document the general process HERE.
+#### Python Script
+##### Requirements:
+
+- Python version: 2.7
+
+##### Installation instructions:
+  1. Download the repo
+  2. Create the file credentials.py into config/
+    * There is a file called credentials.py.example copy that one into that folder remove the .example extension.
+    * Modify the file to add your credentials, and set the name of the report
+  3. Install the dependencies run:
```

Comment:
The user might not have pip installed in their operating system. You should add the instructions to add the pip installer system wide.